### PR TITLE
testing/lib_users: new aport

### PR DESCRIPTION
**Pull request by uu**
Check for mapped libs and open files that are marked as deleted

https://github.com/klausman/lib_users

### CODE REVIEW

File: testing/lib_users/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+pkgname=lib_users
+pkgver=0.13
+pkgrel=0
+pkgdesc="Checks /proc for libraries being mapped but marked as deleted"
+url="https://github.com/klausman/lib_users"
+arch="noarch"
+license="GPL-2.0-or-later"
+options="!check"
+makedepends="python2-dev python3-dev py-setuptools"
+subpackages="py2-${pkgname#py-}:_py2 py3-${pkgname#py-}:_py3"
```
Comment (maxice8) [inline testing/lib_users/APKBUILD]: please don't add python2

File: testing/lib_users/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+# Maintainer: Michael Pirogov <vbnet.ru@gmail.com>
+pkgname=lib_users
+pkgver=0.13
+pkgrel=0
+pkgdesc="Checks /proc for libraries being mapped but marked as deleted"
+url="https://github.com/klausman/lib_users"
+arch="noarch"
+license="GPL-2.0-or-later"
+options="!check"
+makedepends="python2-dev python3-dev py-setuptools"
```
Comment (maxice8) [inline testing/lib_users/APKBUILD]: only py3-setuptools is required

File: testing/lib_users/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# Contributor: Michael Pirogov <vbnet.ru@gmail.com>
+# Maintainer: Michael Pirogov <vbnet.ru@gmail.com>
+pkgname=lib_users
```
Comment (maxice8) [inline testing/lib_users/APKBUILD]: pkgname should be py3-lib_users

File: testing/lib_users/APKBUILD (hunk 1 above)
Comment (uu) [reply testing/lib_users/APKBUILD]: removed.

File: testing/lib_users/APKBUILD (hunk 2 above)
Comment (uu) [reply testing/lib_users/APKBUILD]: done

File: testing/lib_users/APKBUILD (hunk 3 above)
Comment (uu) [reply testing/lib_users/APKBUILD]: renamed:)

File: testing/lib_users/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+pkgver=0.13
+pkgrel=0
+pkgdesc="Checks /proc for libraries being mapped but marked as deleted"
+url="https://github.com/klausman/lib_users"
+arch="noarch"
+license="GPL-2.0-or-later"
+options="!check"
```
[the remark this answers is not in the document]
Comment (uu) [reply testing/lib_users/APKBUILD]: bring tests in live:)

**Comment by maxice8** [comment]
squash into a single commit

File: testing/py3-lib_users/APKBUILD [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+_py() {
+	local python="$1"
+	pkgdesc="$pkgdesc ${python#python}"
+	depends="$depends $python"
+	install_if="$pkgname=$pkgver-r$pkgrel $python"
+
+	cd "$builddir"
+	$python setup.py install --prefix=/usr --root="$subpkgdir"
```
Comment (maxice8) [inline testing/py3-lib_users/APKBUILD]: just move this to `package()`

File: testing/py3-lib_users/APKBUILD [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+_py3() {
```
Comment (maxice8) [inline testing/py3-lib_users/APKBUILD]: remove this

File: testing/py3-lib_users/APKBUILD [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+depends="python3"
+makedepends="py3-setuptools"
+source="$pkgname-$pkgver.tar.gz::https://github.com/klausman/lib_users/archive/v$pkgver.tar.gz"
+builddir="$srcdir"/$_pkgname-$pkgver
+
+build() {
+	python3 setup.py build
+}
+
+package() {
```
Comment (maxice8) [inline testing/py3-lib_users/APKBUILD]: this should be just the python3 setup.py install command from the _py() function

**Pull request closed without merging**